releasenotes/acs-calling-android-sdk-release-notes.md

```diff
@@ -1,11 +1,21 @@
 # ACS Calling Android (Java) SDK - Release History
 
+## v2.2.0-beta.1 (2022-05-13)
+## New Features:
+- Push Notifications support for stopping an incoming call because it was answered in another device, or caller cancelled, etc.
```

Comment:
Would this be wording?
```suggestion
- Push Notifications support for stopping an incoming call because it was answered in another device, or cancelled by the caller, etc.
```

Reply:
Let's update "answered in another device" to "answered by another device" as well.